webpack/scenes/Subscriptions/SubscriptionActions.js

```diff
@@ -76,6 +80,27 @@ export const updateQuantity = (quantities = {}) => (dispatch) => {
     .catch(result => dispatch(apiError(UPDATE_QUANTITY_FAILURE, result)));
 };
 
+export const loadTableColumns = selectedColumns => (dispatch) => {
+  const enabledColumns = (selectedColumns ?
```

Comment:
You could use

```js
export const loadTableColumns =
  (enabledColumns = SUBSCRIPTION_TABLE_DEFAULT_COLUMNS) => (dispatch) => { ... };
```

Reply:
I updated this one a little bit.  Though I normally agree with your suggestion, this particular use case I ended up going with a slightly different solution.